apps/pom.xml

```diff
@@ -298,12 +300,17 @@
 
 			<dependency>
 				<!-- The test framework used for unit and integration tests. -->
-				<groupId>junit</groupId>
-				<artifactId>junit</artifactId>
-				<version>4.13.1</version>
+				<groupId>org.junit.jupiter</groupId>
+				<artifactId>junit-jupiter-engine</artifactId>
+				<version>${junit.version}</version>
+				<scope>test</scope>
+			</dependency>
+			<dependency>
+				<groupId>org.junit.jupiter</groupId>
+				<artifactId>junit-jupiter-params</artifactId>
+				<version>${junit.version}</version>
 				<scope>test</scope>
```

Comment:
can you add a comment on this dependency (i.e. 'parametrized test extension for junit' if that is what it is)